pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -464,8 +464,24 @@ func (s *Service) GetMigrationRunList(ctx context.Context, migUID string) (*clou
 	return runList, nil
 }
 
-func (s *Service) DeleteSession(ctx context.Context, uid string) (*cloudmigration.CloudMigrationSession, error) {
-	c, err := s.store.DeleteMigrationSessionByUID(ctx, uid)
+func (s *Service) DeleteSession(ctx context.Context, sessionUID string) (*cloudmigration.CloudMigrationSession, error) {
```

Comment:
Can we delete the snapshot files as well since they won't be used anymore and are occupying disk space?

Reply:
done!